lib/api/client-commands/network/captureNetworkRequests.js

```diff
@@ -9,7 +9,7 @@ const {Logger} = require('../../utils');
  *    it('captures and logs network requests as they occur', function(this: ExtendDescribeThis<{requestCount: number}>) {
  *      this.requestCount = 1;
  *      browser
- *        .captureNetworkRequests((requestParams) => {
+ *        .network.captureNetworkRequests((requestParams) => {
```

Comment:
It should be `.network.captureRequests` here. Similar for other two files as well.